ansible_collections/arista/avd/roles/eos_designs/docs/tables/l3-edge.md

```diff
@@ -317,4 +344,83 @@
           # Custom structured config for interfaces
           # Note! The content of this dictionary is _not_ validated by the schema, since it can be either ethernet_interfaces or port_channel_interfaces.
           structured_config: <dict>
+      l3_interfaces_profiles:
+
+          # L3 interface profile name. Any variable supported under `l3_interfaces` can be inherited from a profile.
+        - name: <str; required; unique>
+
+          # Speed should be set in the format `<interface_speed>` or `forced <interface_speed>` or `auto <interface_speed>`.
+          speed: <str>
+
+          # Node IPv4 address/Mask or 'dhcp'.
+          ip: <str>
+
+          # Supported if `ip` is `dhcp`.
+          # Accepts default route from DHCP, default False.
+          dhcp_client_accept_default_route: <bool>
+
+          # Must be an Ethernet interface like 'Ethernet2'.
+          interface: <str>
+
+          # The peer device name. Used for description and documentation
+          peer: <str>
+
+          # The peer device interface. Used for description and documentation
+          peer_interface: <str>
+
+          # The peer device IP. Used for description and documentation
+          peer_ip: <str>
+
+          # Interface description.
+          description: <str>
+
+          # QOS service profile.
+          qos_profile: <str>
+
+          # EOS CLI rendered directly on the interface in the final EOS configuration.
+          raw_eos_cli: <str>
+
+          # Custom structured config for interfaces
+          structured_config: <dict>
+      l3_interfaces:
+
+          # Device on which the interface should be configured.
+        - node: <str>
+
+          # L3 interface profile name. Profile defined under l3_interfaces_profiles.
+          profile: <str>
+
+          # Speed should be set in the format `<interface_speed>` or `forced <interface_speed>` or `auto <interface_speed>`.
+          speed: <str>
+
+          # Node IPv4 address/Mask or 'dhcp'.
+          ip: <str>
+
+          # Supported if `ip` is `dhcp`.
+          # Accepts default route from DHCP, default False.
+          dhcp_client_accept_default_route: <bool>
+
+          # Must be an Ethernet interface like 'Ethernet2'.
+          interface: <str>
+
+          # The peer device name. Used for description and documentation
+          peer: <str>
+
+          # The peer device interface. Used for description and documentation
+          peer_interface: <str>
+
+          # The peer device IP. Used for description and documentation
+          peer_ip: <str>
+
+          # Interface description.
```

Comment:
Move this to the top - just below interface.

Also `# Interface description. If not set a default description will be configured with '[<peer>[ <peer_interface>]]'`.